src/backend/core/api/serializers.py

```diff
@@ -32,21 +32,10 @@ class Meta:
 class UserLightSerializer(UserSerializer):
     """Serialize users with limited fields."""
 
-    id = serializers.SerializerMethodField(read_only=True)
-    email = serializers.SerializerMethodField(read_only=True)
-
-    def get_id(self, _user):
-        """Return always None. Here to have the same fields than in UserSerializer."""
-        return None
-
-    def get_email(self, _user):
-        """Return always None. Here to have the same fields than in UserSerializer."""
-        return None
-
     class Meta:
         model = models.User
-        fields = ["id", "email", "full_name", "short_name"]
-        read_only_fields = ["id", "email", "full_name", "short_name"]
+        fields = ["full_name", "short_name"]
+        read_only_fields = ["full_name", "short_name"]
```

Comment:
I did that to avoid the front application to have multiple models.

Reply:
The question it raised was: how do I differentiate between an model who has the value and the value is None and a model that doesn't have it? 
I removed it because I was introducing more differences between the 2 serializers and it would be confusing to have these other fields as well with null values :thinking: